tests/designs/axi4_ram/Makefile

```diff
@@ -0,0 +1,33 @@
+# Copyright cocotb contributors
+# Licensed under the Revised BSD License, see LICENSE for details.
+# SPDX-License-Identifier: BSD-3-Clause
+
+TOPLEVEL_LANG ?= verilog
+
+ifneq ($(TOPLEVEL_LANG),verilog)
+
+all:
+	@echo "Skipping test due to TOPLEVEL_LANG=$(TOPLEVEL_LANG) not being verilog"
+clean::
+
+else
+
+TOPLEVEL := top
+
+PWD=$(shell pwd)
+
+COCOTB?=$(PWD)/../../..
+
+VERILOG_SOURCES = \
```

Comment:
Personally, I like to append multiple lines using ``VERILOG_SOURCES += ...`` repeatedly because it enables adding comments in between, but that's a matter of preference.